news/_posts/2020-04-27-Polymake.jl-0.4-has-been-tagged.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Polymake.jl 0.4 has been tagged
+author: Marek Kaluba
+---
+
+`Polymake.jl` v0.4.0 has been tagged!
+
+Since we're upgrading to support `CxxWrap.jl` `v0.10.1`, we decided to bump the major 
+version to force compilation of libpolymake against libcxxwrap-julia. 
+The release has been somewhat hastened by the need for `Oscar.jl` jointly working with 
+the newest `CxxWrap.jl`, so expect `v.0.4.1` and later minor versions rather heavy
+in added features :)
+
+<br/>
```

Comment:
Why the explicit line break?

Reply:
Without, I found it looks too dense, and introducing a new paragraph looks nicer.